# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323258912 18000
#      Wed Dec 07 06:55:12 2011 -0500
# Node ID 54f76558c41a8974290931bb80923cb83f83e811
# Parent  f8556baf1949ad7662557e549906ebf2d6188ea1
avoid gh_manager segfault

* graphics.cc (gh_manager::create_instance): Don't register cleanup
function.

diff --git a/src/graphics.cc b/src/graphics.cc
--- a/src/graphics.cc
+++ b/src/graphics.cc
@@ -7396,17 +7396,19 @@ gh_manager::gh_manager (void)
   graphics_toolkit::default_toolkit ();
 }
 
 void
 gh_manager::create_instance (void)
 {
   instance = new gh_manager ();
 
+#if 0
   singleton_cleanup_list::add (cleanup_instance);
+#endif
 }
 
 graphics_handle
 gh_manager::do_make_graphics_handle (const std::string& go_name,
                                      const graphics_handle& p,
                                      bool integer_figure_handle,
                                      bool do_createfcn,
                                      bool do_notify_toolkit)
